updating/preparing_for_updates/updating-cluster-prepare.adoc

```diff
@@ -15,8 +15,12 @@ To do: Remove this comment once 4.13 docs are EOL.
 Learn more about administrative tasks that cluster admins must perform to successfully initialize an update, as well as optional guidelines for ensuring a successful update.
 
 [id="kube-api-removals_{context}"]
-== Kubernetes API deprecations and removals
+== Kubernetes API removals
 
+There are no Kubernetes API removals in {product-title} 4.15.
```

Comment:
If this change is for 4.15 upgrade, we should change `Preparing to update to {product-title} 4.14` to 4.15. Just like what we did in 4.14 https://docs.openshift.com/container-platform/4.14/updating/preparing_for_updates/updating-cluster-prepare.html

Reply:
Good catch @wangke19, updated the titles to be 4.15.